src/avt/VisWindow/Colleagues/avtTimeSliderColleague.h

```diff
@@ -36,6 +36,8 @@ class vtkTimeSliderActor;
 //   Brad Whitlock, Wed Sep 28 15:24:47 PDT 2011
 //   Change the text actor type.
 //
```

Comment:
Move this blank line after the comment.

Reply:
Hmmm...totally confused by this review comment.  Line 38 is blank and is below a comment. And all that was there before. I think you mean you want a blank comment line *after* the modification comment that I added and am committing a changed for that.

Reply:
Yes, a blank line after the comment. Sorry I wasn't clear. I didn't know I could make multiline suggestions. I'll try that next time.